ruby-stubs/r10k-validate

```diff
@@ -0,0 +1,14 @@
+require 'English'
+
+status = 0
+ARGV.each do |file|
+  output = `( cd $(dirname "#{file}" && r10k puppetfile check 2>&1)`
```

Comment:
Oops, just realized -- I think there's a missing end parenthesis after the `dirname "#{file}"`? I get a syntax error when I test out this hook.

Reply:
Fixed.